src/App.jsx

```diff
@@ -1,18 +1,12 @@
 import React from 'react';
-import { useState } from 'react'
 import Calculator from './Components/Calculator'
-import './App.css'
 
-class App extends React.Component {
-  constructor(props) {
-    super(props)
-  }
 
-  render(){
+function App(){
     return(
```

Comment:
- [ ] Kindly use arrow function instead of conventional functions to follow ES6 😊

Your Function 🔴
```
function App(){
// your contents
}
```

Arrow Function 🟢
```
const App = () => {
// your contents
}
```